design/additional-data.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Additional fields synchronization
+  - As a store owner
+  - I want to get subscribers details from store
+  - So I would be able to send personalized emails and filter subscribers
```

Comment:
>Feature description's "feature" part is not concrete enough.

Good point. I am thinking something like this may be the solution. I would keep the email part in **benefit** section though

 >As a store owner
 I want to synchronize subscribers´ additional details to Smaily
 So I can send personalized emails and target subset of my subscribers

Reply:
We started using "message" instead of "email" because people would shorten "email address" to "email", and it wasn't always clear what was meant in the context of a sentence.

Reply:
Okay, I will change it here also.

Reply:
Changed in https://github.com/sendsmaily/Sendsmaily-Sync-for-Magento/pull/8/commits/e2154ff71c63e7aa80195730e28a746bc41dbd6e